js/web/package-lock.json

```diff
@@ -91,9 +91,9 @@
       }
     },
     "node_modules/@electron/get": {
-      "version": "1.13.1",
```

Comment:
It probably is not a bad idea to upgrade the package version, but should not in this PR.